skopt/acquisition.py

```diff
@@ -0,0 +1,203 @@
+import warnings
+
+import numpy as np
+
+from scipy.stats import norm
+
+
+def gaussian_acquisition(X, model, y_opt=None, method="LCB",
```

Comment:
But then we will have to write a `if else` block for every new function added. Are you okay with that?